hasgeek/funnel
# Add e2e tests

1. Add test to verify adding livestream to project
2. Update schedule edit test to include
   a) adding confirmed proposal to schedule
   b) verifying present of colour palette dropdown for venue rooms.
3. Add test to verify list of attendees(rsvp list) who have responded yes to attending a project.
4. Change wait timer of login to wait for login xhr response.
